src/Agent.Worker/Release/ContainerFetchEngine/ContainerFetchEngineOptions.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Threading;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release.ContainerFetchEngine
+{
+    public class ContainerFetchEngineOptions
+    {
+        public CancellationToken CancellationToken { get; set; }
```

Comment:
Yeah, I can have it in constructor as an optional parameter. Will do it as a part of later refactoring.